src/lib/components/home/section_title.tsx

```diff
@@ -1,18 +1,14 @@
 import * as React from "react"
 
-import {
-  StyleSheet,
-  Text,
-  TextStyle,
-} from "react-native"
+import { StyleSheet, Text, TextStyle } from "react-native"
 
 export default class SectionTitle extends React.Component<any, any> {
   static propTypes = {
     ...Text.propTypes,
   }
 
   render() {
-    const { children, ...props } = this.props
+    const { children, props } = this.props
```

Comment:
Oh wow, this is bad, how come the AST check didn't catch that! Could you create an issue?